inference-engine/src/mkldnn_plugin/nodes/mkldnn_extract_image_patches_node.h

```diff
@@ -62,15 +64,65 @@ class MKLDNNExtractImagePatchesNode : public MKLDNNNode {
     std::vector<size_t> _ksizes;
     std::vector<size_t> _strides;
     std::vector<size_t> _rates;
-    size_t _pad_left;
-    size_t _pad_top;
-    std::shared_ptr<jit_uni_extract_image_patches_kernel> extract_image_patches_kernel;
     static const std::set<size_t> _supported_precisions_sizes;
-
     ExtImgPatcherPadType _auto_pad;
-    InferenceEngine::Precision precision;
 
     std::string errorPrefix;
+
+    struct ExtractImagePatchesExecutor {
+        ExtractImagePatchesExecutor() {}
+        virtual void exec(void* src, void* dst, const VectorDims& istrides, const VectorDims& ostrides) = 0;
+        jit_extract_image_patches_params fillJpp(
```

Comment:
add `private` section in `ExtractImagePatchesExecutor`
the same for others Executors

Reply:
`protected` section was added for the base class (because the fields should be accessible from subclasses) and `private` - for subclasses. Thank you!